l10n_th_vendor_tax_invoice/wizard/wizard_tax_adjustments.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class TaxAdjustments(models.TransientModel):
+    _inherit = 'tax.adjustments.wizard'
+
+    debit_partner_id = fields.Many2one(
+        string='Partner (Debit Account)',
+        comodel_name='res.partner',
+    )
+    credit_partner_id = fields.Many2one(
+        string='Partner (Credit Account)',
+        comodel_name='res.partner',
+    )
+    tax_date = fields.Date()
+    tax_invoice = fields.Char(
+        string='Tax Invoice Number',
+    )
+    amount_tax_base = fields.Monetary(
+        string='Tax Base',
+        currency_field='company_currency_id',
+        required=True,
+    )
+
+    @api.one
+    @api.constrains('amount', 'amount_tax_base')
+    def _check_amount(self):
+        if not self.amount or not self.amount_tax_base:
+            raise ValidationError(_(
+                'Tax Amount or Tax Base can\'t have a 0 amount.'))
+
+    @api.multi
+    def _create_move(self):
+        res = super()._create_move()
```

Comment:
```suggestion
        move_id = super()._create_move()
```
I think it will be more readable/understandable.